### PR TITLE
Us89544   refresh course image upload

### PR DESCRIPTION
**Pull request by capajon**

### CODE REVIEW

**Comment by ryantmer** [comment]
You'll probably want to rebase or merge master - I just merged my `d2l-fetch` changes which _shouldn't_ have affected this at all, but might be worth making sure

File: d2l-course-tile-grid.html [hunk 1]
```diff
@@ -122,22 +122,35 @@
...
 						organization && organization.properties &&
 						this.getEntityIdentifier(courseTileOrg) === this.getEntityIdentifier(organization)
 					) {
-						courseTiles[x].focus();
-						return true;
+						return courseTiles[x];
 					}
 				}
-				return false;
+				return;
```
[the remark this answers is not in the document]
Comment (capajon) [reply d2l-course-tile-grid.html]: I know, I suppose I wanted to be explicit, somehow it felt right in the moment. ;)  Might change.

File: d2l-course-tile.html [hunk 2]
```diff
@@ -284,6 +282,24 @@
...
 			},
+			// Refreshes image from organization API call (especially useful when uploading course image)
+			refreshImage: function() {
+				var tileContainer = this.$$('.tile-container');
+
+				this._forceImageRefresh = true;
+				this.toggleClass('change-image-loading', true, tileContainer);
+
+				this._fetchOrganization()
+					.then(
```
[the remark this answers is not in the document]
Comment (capajon) [reply d2l-course-tile.html]: Yeah I like that... I don't often think to bind arguments.  That is much cleaner looking. :)  Will change!

[the remark this answers is not in the document]
Comment (ryantmer) [reply d2l-course-tile.html]: I find it's super useful in situations like this, where you're chaining a bunch of stuff via Promises :D

**Pull request merged**